CuratedContent/ScalingYourselfSoftwareDeveloper.md

```diff
@@ -0,0 +1,28 @@
+# Scaling Yourself as a Software Developer 
+
+<!-- deck text start -->
+Do you find yourself developing software for your main projects outside of regular work time just to look or be productive? Do interruptions from email, social media, or requests from others cause your day to slip away? This lively and very helpful video gives numerous ideas to help software developers (or anyone) to "scale" personal productivity.
+
+<!-- deck text end -->
+
+#### Contributed by [Patricia  Grubel](https://github.com/pagrubel "Patricia Grubel Github Profile")
+#### Publication date: March 13, 2021 
+
+Resource information | Details
+:--- | :---
+Website  | [Scaling Yourself as a Software Developer](http://blog.martinig.ch/videos/scaling-yourself-as-a-software-developer/)
+Author |Scott Hanselman
+Focus | Productivity for software developers
+
+
+In the video *[Scaling Yourself as a Software Developer](http://blog.martinig.ch/videos/scaling-yourself-as-a-software-developer/)*,
+Scott Hanselman helps software developers (and others) prioritize productive work, minimize interruptions (by self and others), have productive work sprints, and organize and schedule necessary but possibly time-consuming tasks such as email. He includes suggestions for planning your day and beyond so that you don't become entangled or driven by other factors. After watching this video and implementing a few of his recommendations, you may find yourself breaking an "urgency" addiction and moving towards real productivity. At the very least your inbox will be cleared out quickly and the number of open browser windows will shrink. Productivity ideas and tools are suggested in the video such as using a personal kanban and occasionally syncing to paper. The video also suggests tools you can use to organize activities that might interrupt your productive software development time, but you would like to do later, such as using a service like *Instapaper* to organize sites to read later.  The video is entertaining and may even prompt me to get a pomodoro; watch the video to find out why. 
```

Comment:
Can you split this into multiple lines of approx 80-120 chars each? I don't think bssw.io EB team has agreed upon a line length yet but I think we all agree *some reasonable* line length is a good thing.

I wanted to suggest you break this single paragraph into 2 or 3. I could easily suggest *where* too if it was multiple lines.

I'd start a new paragraph at `After watching this video...` and maybe also at `The video also suggests tools...`

Reply:
Thanks @markcmiller86, I believe I implemented all your suggestions, but please look over it for anything else or for any mistakes the changes may have introduced.